cx_Freeze/freezer.py

```diff
@@ -39,7 +40,20 @@
 __all__ = ["ConfigError", "ConstantsModule", "Executable", "Freezer"]
 
 
-class Freezer:
+class Freezer(ABC):
+    def __new__(cls, *args, **kwargs):
+        # create instance of appropriate sub-class, depending on the platform.
+        instance: Freezer
+        if sys.platform == "win32":
+            instance =  super().__new__(WinFreezer)
+        elif sys.platform == "darwin":
+            instance =  super().__new__(DarwinFreezer)
+        elif sys.platform == "linux":
```

Comment:
In the current code, when it is not Windows or macOS, then it is Linux. So, let's consider it this way because those who use Unix, FreeBSD, VxWorks must be able to build their application without any problems using the rules for Linux.

Reply:
Okay, makes sense. Fixed.